packages/mdc-segmented-button/segment/_segment.scss

```diff
@@ -77,28 +126,57 @@
   }
 }
 
+@mixin ripple($query: feature-targeting.all()) {
```

Comment:
Move this to `_segment-ripple.scss` file and include with and without ripple styles in `/segment/styles.scss`